Update reqwest to be compatible to openssl 1.1.1

schaeferpp opened the pull request:
The specified `reqwest` version fails to build with the recent openssl version (tested on Arch Linux).
Allow using newer 0.9 versions with this commit.

`cargo test` does not report any errors.

Commit:
Update reqwest to be compatible to openssl 1.1.1

jonas-schievink commented:
This change doesn't do anything AFAICT. You just have to run `cargo update` on your project to get a newer version.

schaeferpp commented:
Sorry for the noise. Updated my `Cargo.toml` to use your git version instead of the [crates.io one](https://crates.io/crates/xmlrpc). I did not expect there to be differences as the version on crates.io matches this version.

Maybe that is the main problem and I addressed the wrong one. Crates.io lists the dependency `reqwest ^0.8.0` instead of `reqwest 0.9.0`.

jonas-schievink commented:
Yes, sorry, I should've released 0.13.0 a while ago. I'll do that now.

schaeferpp commented:
Thank you!

jonas-schievink commented:
Okay, that should do. If you still run into any problems just open an issue!
